terraform/groups/ecs-service/.gitignore

```diff
@@ -0,0 +1 @@
+.terraform*
```

Comment:
I think this entry needs to go into the .gitignore file in the root of thus git repo.